src/site/content/en/blog/async-clipboard/index.md

```diff
@@ -209,6 +209,23 @@ async function getClipboardContents() {
 }
 ```
 
+### Working with pasted files
```

Comment:
Are you implying that simply implementing copy, cut, and pate events will make the keyboard shortcuts function? You should say that if such is the case.

Reply:
I have clarified this.